data/hai/unfettered side missions.txt

```diff
@@ -769,11 +769,12 @@ mission "<uhai> 20: Boarding Equipement: More Enforcer Staffs"
 		<staff> "Enforcer"
 	to offer
 		has "<uhai> 20: Bigeminate Sigil: Boarding Equipement: offered"
-		not "Bring Coalition Weapons: offered"
-		not "Bring Pug Staffs: offered"
+		not "<uhai> 20: Boarding Equipement: Enforcer Staffs: offered"
+		not "<uhai> 20: Boarding Equipement: Pug Staffs: offered"
 	on offer
 		require "Enforcer Riot Staff"
 		conversation "more staffs"
+		fail
```

Comment:
The convo ends with declining by default so this isnt required
Cant hurt for clarity tho